### DISCUSSION TITLE
deep query with M2A relationship doesn't work

### DISCUSSION THREAD

**Discussion by danielefocarelliagic** [directus/directus#16411] [Archive]
### Describe the Bug

When I try to select the translation with a deep query on the M2A data model, it does't work

### To Reproduce

I created a data model like this:

![MicrosoftTeams-image (7)](https://user-images.githubusercontent.com/117266390/201147737-d415f9e0-e4dd-4020-9dfb-31d8ea577163.png)

![MicrosoftTeams-image (8)](https://user-images.githubusercontent.com/117266390/201147800-aebe7a1a-f685-41f1-9cd7-7703005519e7.png)

then I try to select the translation with a deep query like this:

![MicrosoftTeams-image (9)](https://user-images.githubusercontent.com/117266390/201148224-f9baf627-b853-4df2-8d1d-64e6fc2deb12.png)

It return both translations instead of en-US.

### What version of Directus are you using?

9.20.3

### What version of Node.js are you using?

16.18.0

### What database are you using?

SQLite 

### What browser are you using?

Chrome

### How are you deploying Directus?

PaaS

**Comment by licitdev** [comment]
The deep query looks to be invalid.
The filtering of m2a fields require the collection scope to be provided. Ref: https://docs.directus.io/reference/query.html#many-to-any-union-types
In addition, the `translations` field is missing from your deep query.

Please test with `deep[components][item:labelAndDescription][translations][_filter][languages_code][_eq]`.

**Reply by danielefocarelliagic** [reply]
Thanks for the reply, but it still doesn't work

![MicrosoftTeams-image (10)](https://user-images.githubusercontent.com/117266390/201315547-2d7a08b9-45ca-43f3-877e-3be393b9411b.png)

**Reply by licitdev** [reply]
Realised a typo, does removing the spacing after `item: ` work?

**Reply by danielefocarelliagic** [reply]
Great, it's work. But I have another question. is It possible to insert a wildcard "collection-scope" like this: 

![MicrosoftTeams-image (11)](https://user-images.githubusercontent.com/117266390/201317944-638007b2-2a96-48ff-908b-0eaa6ac2e55d.png)

**Reply by licitdev** [reply]
Good to hear that it works.
Wildcard "collection-scope" isn't supported currently. 
Feel free to start a separate discussion on the idea. Thanks! 👍
